config/locales/en.yml

```diff
@@ -130,6 +130,15 @@ en:
         option_no:
           label: "No"
       custom_select_error: Select which services you can provide
+    offer_care:
+      title: Can you offer social care or childcare?
+      hint: For example, nursing or assistance to other people.
+      options:
+        option_yes:
+          label: "Yes"
+        option_no:
+          label: "No"
+      custom_select_error: Select which services you can provide
```

Comment:
Does this make sense as a boolean form?

Reply:
Good question, will put out a shout in slack for error message copy:
Just stole this one for consistency for now: https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/blob/master/config/locales/en.yml#L132

Reply:
Asked here: https://gds.slack.com/archives/C010FS40HHD/p1584788365118600